Dockerfile

```
@@ -1,14 +1,14 @@
 FROM ubuntu:focal-20211006
 
-ARG VERSION=14.4.2
+ARG VERSION=14.4.4
 
 ENV GITLAB_VERSION=${VERSION} \
     RUBY_VERSION=2.7.4 \
     RUBY_SOURCE_SHA256SUM="3043099089608859fc8cce7f9fdccaa1f53a462457e3838ec3b25a7d609fbc5b" \
```

GitLab omnibus package updated ruby to v2.7.5 for security concern. Do we follow it?
https://gitlab.com/gitlab-org/omnibus-gitlab/-/commit/a236f58e72d1e8cdb8642a63c0631b19ac78b9a8
https://about.gitlab.com/releases/2021/12/06/security-release-gitlab-14-5-2-released/#update-ruby

ruby release notes:
https://www.ruby-lang.org/en/news/2021/11/24/ruby-2-7-5-released/

```suggestion
    RUBY_VERSION=2.7.5 \
    RUBY_SOURCE_SHA256SUM="2755b900a21235b443bb16dadd9032f784d4a88f143d852bc5d154f22b8781f1" \
```

---

Hi,

thanks for your hint. I'll add ruby 2.7.5 to this PR.